### ISSUE TITLE
Murkmod Helper extension "Manifest version 2 is deprecated"

### ISSUE THREAD

**Issue by Eboi47** [rainestorme/murkmod#97]
**Describe the bug**
The murkmod helper extension will say mush is not ready yet if you try to use anything. The mush terminal is still functional. I believe it is tied to the error which appears when viewing the extension page.

**To Reproduce**
1. Install the murkmod helper extension
2. Click on the puzzle piece, and then the murkmod helper extension
3. Click any button

**Expected behavior**
The button would be pressed and the task would be successfully executed.

**Screenshots**
![Screenshot 2024-10-17 10 31 17 AM](https://github.com/user-attachments/assets/eba6fd09-bb72-4bd0-a752-6a7f88b1761b)
![Screenshot 2024-10-17 10 25 17 AM](https://github.com/user-attachments/assets/75c55a86-0fc0-4652-93f7-12c98d26d5c0)

**System (please complete the following information):**
 - Board [dedede]
 - Version [v119]
- Installation method used [devmode installer]

**Additional context**
When viewing the extension in the chrome web store, it shows up as the deprecated Secure Shell extension. The non-deprecated version is [here](https://chrome.google.com/webstore/detail/iodihamcpbpeioajjeobimgagajmlibd)

**Comment by CaenJones** [comment]
As of this summer Google has switched to manifestV3 and has depreciated the manifestv2 standard. Because the manifest file has not been updated, the extension will not work.

https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline

**Comment by rainestorme** [comment]
> As of this summer Google has switched to manifestV3 and has depreciated the manifestv2 standard. Because the manifest file has not been updated, the extension will not work.
> 
> https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline

Yep, that makes sense. The trick that allows the helper extension to work is that the keys are exposed in the ChromiumOS source tree... is there an alternative in newer versions for MV3?

**Comment by rainestorme** [comment]
You can see a list of the keys for extensions pretty easily in codesearch [here](https://source.chromium.org/search?q=file:manifest.json%20%22key%22:&sq=&ss=chromiumos%2Fchromiumos%2Fcodesearch) - I'm sure at least one of these extensions would work - or maybe the current extension can just be updated straight to MV3? Who knows, this would need testing, and I don't have any testing devices on-hand right now.

**Comment by wiiuavi** [comment]
was a fix for this ever found?

**Comment by rainestorme** [comment]
> was a fix for this ever found?

Not that I know of. I welcome testing of porting to MV3 though.
